### PR TITLE
DOC-3184: Broken link in CE 4.5 release notes

### PR DESCRIPTION
**Pull request by AaronForinton**
The link to the Cryptoservice Configuration page needs to be to https://docs.corda.net/docs/corda-enterprise/4.5/node/operating/cryptoservice-configuration.html-

It is currently linking to https://docs.corda.net/docs/corda-enterprise/4.5/cryptoservice-configuration.md (note the .md extension in the link definition that needs fixed).

### CODE REVIEW

**Comment by AaronForinton** [comment]
Screenshot 2021-11-22 at 15 11 25
This is the relevant change.

File: content/en/platform/corda/4.5/enterprise/release-notes-enterprise.md [hunk 1]
```diff
@@ -253,7 +253,7 @@ The documentation has been relocated to the main Corda and Corda Enterprise docu
 ### Other improvements
 
 * All database columns containing datestamps have been standardised to use UTC (the time zone used was previously inconsistent).
-* The HSM name used in the HA Utilities `--bridge-hsm-name` and `--float-hsm-name` command-line parameters should now exactly match `cryptoServiceName`, as described [here](../../../../../en/platform/corda/4.5/enterprise/cryptoservice-configuration.md).
+* The HSM name used in the HA Utilities `--bridge-hsm-name` and `--float-hsm-name` command-line parameters should now exactly match `cryptoServiceName`, as described [here](../../../../../en/platform/corda/4.5/enterprise/cryptoservice-configuration.html).
```
Comment (michelemarsden) [inline content/en/platform/corda/4.5/enterprise/release-notes-enterprise.md:256]: The .md is fine (not a link to a header). The file is in a different folder:
```suggestion
* The HSM name used in the HA Utilities `--bridge-hsm-name` and `--float-hsm-name` command-line parameters should now exactly match `cryptoServiceName`, as described [here](../../../../../en/platform/corda/4.5/enterprise/node/operating/cryptoservice-configuration.md).
```

**Review by michelemarsden — COMMENTED** [review]
See my comment.

**Review by michelemarsden — APPROVED** [review]
LGTM

**Pull request merged**